### ISSUE TITLE
Agenda

### ISSUE THREAD

**Issue by adamhenriques** [Praqma/code-conf.com#342]
Could we design an agenda design for our conferences? Next event is Aug 25th where we need something.

Keep in mind that most of our events run with parallel track - starting with a keynote in plenum and then braching out in two tracks.

previous design
http://archive.code-conf.com/day-of-docker-osl15/agenda/

Another design option;
http://deliveryofthingsworld.com/en/agenda/

**Comment by illus0r** [comment]
@adamhenriques 
Both design and implementation have different sizes. To estimate technical side I need @jonnycoull to make a mockup first.

**Comment by jonnycoull** [comment]
@adamhenriques @illus0r if you cast your minds back we did this a while ago, I think its been lost in the issues. looked like this, remember @illus0r ? (OLD MOCK UP)
![screen shot 2016-08-16 at 15 58 43](https://cloud.githubusercontent.com/assets/17589853/17704531/33cfaf16-63cd-11e6-93cf-1673b916827b.png)

**Comment by jonnycoull** [comment]
minus the print bit.

**Comment by adamhenriques** [comment]
@jonnycoull i like it! Will it be custamizable Per event?

**Comment by illus0r** [comment]
@adamhenriques 
Hi there, Adam! 
Can we have up to 3 branches (line on http://archive.code-conf.com/day-of-docker-osl15/agenda/)?

**Comment by illus0r** [comment]
@adamhenriques 
Can some events in parallel branches start in different time?
line that
```
+-------------+    +------------+
|             |    |  event3    |
|   event1    |    +------------+
|             |
+-------------+    +------------+
                   |            |
+--------------+   |            |
|              |   |  event4    |
|   e^ent2     |   |            |
+--------------+   +------------+
```

**Comment by illus0r** [comment]
@adamhenriques 
or like that
```
+-------------+   +------------+
|             |   |            |
|   event1    |   |            |
|             |   |            |
+-------------+   |            |
                  |   event3   |
+--------------+  |            |
|              |  |            |
|   e^ent2     |  |            |
+--------------+  +------------+
```

**Comment by adamhenriques** [comment]
@illus0r presentations will always start at the same time. And we nned the possibility to merge the tracks to 1 common branch for keynotes like; https://docs.google.com/spreadsheets/d/1KBpnHxoLUY03iN8Jy-dkT-3OHHPaaTOBgVKjBsIXTA8/edit#gid=0

**Comment by illus0r** [comment]
@adamhenriques 
Will we have triple merges like this one?
![image](https://cloud.githubusercontent.com/assets/797993/17809819/c55157b8-6621-11e6-8a86-0fcdb3deaaf3.png)

**Comment by adamhenriques** [comment]
@illus0r not in the near future - at least not anything planned

**Comment by adamhenriques** [comment]
@illus0r do you have a status/ eta on this?

**Comment by illus0r** [comment]
I'll move it to a separate milestone, so, not yet. Let me know if you have any objections, we can establish another issue order.

**Comment by illus0r** [comment]
@adamhenriques

**Comment by adamhenriques** [comment]
@illus0r thanks. When can we have this ready? It would be nice for our conferences this and next week.

**Comment by illus0r** [comment]
> Is there any way we can have an agenda format live sometime soon so we can display the conference agenda of the event thursday and coming Monday?

Hi there! So you need it tomorrow?

> Adam Henriques
> Or today?;-)

@adamhenriques can you send me the link to event and agenda you want to put there?

**Comment by adamhenriques** [comment]
@illus0r event; http://www.code-conf.com/doc-sthlm-2016/

Agenda; https://docs.google.com/spreadsheets/d/1nN-OqF217BbLz2_doynMDiC7Bm6DrCx6MnwrhyHvZlk/edit#gid=1790899168

Mockup; https://drive.google.com/a/firm20.com/file/d/0B3mqCwER-jlkQS1DU0M3cnpvVFk/view?usp=sharing

**Comment by adamhenriques** [comment]
@illus0r and for copenhagen;

Event; http://www.code-conf.com/doc-cph-2016/

Agenda; https://docs.google.com/spreadsheets/d/1nN-OqF217BbLz2_doynMDiC7Bm6DrCx6MnwrhyHvZlk/edit#gid=1790899168

Mockup; https://drive.google.com/a/firm20.com/file/d/0B3mqCwER-jlkNWpQM3NCOC1VNms/view?usp=sharing

OBS notice that one is for copenhagen and one for stockholm!

**Comment by illus0r** [comment]
Me
>Hi, mate.
You've drawn this mockup for agenda and I'm about to implement it.
But I see some things that can confuse new participants:
* It looks like I can't jump between branches. But I can.
* It looks like SWAP TRACK, LUNCHBUFFET e.t.c. are for first branch only.

>I'd be happy to spend some time to come up with alternative mockup, which fixes that. I'd make it adapted to mobile. But, sure, I can go on with this mockup, it's good enough. What's your decision?

Bok
> Just take those lines off and that should solve the problem

**Comment by jonnycoull** [comment]
@illus0r duplicate of #400 close?

**Comment by illus0r** [comment]
@jonnycoull milestone 🔫
